src/styles/main.scss

```diff
@@ -0,0 +1,65 @@
+$day-size: 100px;
+$day-gap: 1px;
+$calendar-padding: 10px;
+$calendar-column: 7;
+$calendar-width: $day-size * $calendar-column + $day-gap * ($calendar-column - 1) + 2 * $calendar-padding;
```

Comment:
The calculation of `$calendar-width` is not following the Code Style Rule for math operations. There should be spaces around the `*` and `+` operators to improve readability.